packages/styling/src/styles/UtilityStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { IRawStyle } from '../interfaces/index';
```

Comment:
In all likelihood there will be something to add here, so having a catchall file in the styling folder does make sense.